app/src/main/java/org/jellyfin/androidtv/ui/playback/overlay/action/PlaybackSpeedAction.kt

```diff
@@ -0,0 +1,62 @@
+package org.jellyfin.androidtv.ui.playback.overlay.action
+
+import android.content.Context
+import android.view.Gravity
+import android.view.View
+import android.widget.PopupMenu
+import org.jellyfin.androidtv.R
+import org.jellyfin.androidtv.ui.playback.PlaybackController
+import org.jellyfin.androidtv.ui.playback.VideoSpeedController
+import org.jellyfin.androidtv.ui.playback.overlay.CustomPlaybackTransportControlGlue
+import org.jellyfin.androidtv.ui.playback.overlay.LeanbackOverlayFragment
+import java.util.*
+
+class PlaybackSpeedAction(
+	context: Context,
+	customPlaybackTransportControlGlue: CustomPlaybackTransportControlGlue,
+	playbackController: PlaybackController
+) : CustomAction(context, customPlaybackTransportControlGlue) {
+	private val speedController = VideoSpeedController(playbackController)
+	private val speeds = VideoSpeedController.Companion.SpeedSteps.values()
+
+	init {
+		initializeWithIcon(R.drawable.exo_ic_speed)
```

Comment:
Don't use drawables from libraries, add a new ic_speed to our own drawables instead.